iml-agent/src/agent_error.rs

```diff
@@ -171,6 +173,7 @@ impl std::error::Error for ImlAgentError {
             ImlAgentError::UrlParseError(ref err) => Some(err),
             ImlAgentError::Utf8Error(ref err) => Some(err),
             ImlAgentError::XmlError(ref err) => Some(err),
+            ImlAgentError::LdevEntriesError(ref err) => None,
```

Comment:
```suggestion
            ImlAgentError::LdevEntriesError(_) => None,
```

Clippy is picking up on this